LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/NetworkServerConfiguration.cs

```diff
@@ -154,7 +156,7 @@ public static NetworkServerConfiguration CreateFromEnvironmentVariables()
             var envVars = new CaseInsensitiveEnvironmentVariables(Environment.GetEnvironmentVariables());
             config.ProcessingDelayInMilliseconds = envVars.GetEnvVar("PROCESSING_DELAY_IN_MS", config.ProcessingDelayInMilliseconds);
             config.RunningAsIoTEdgeModule = !envVars.GetEnvVar("CLOUD_DEPLOYMENT", false);
```

Comment:
We can make this row check both CLOUD_DEPLOYMENT and LOCAL_DEVELOPMENT (if any of them is true, the RunningAsIoTEdgeModule will be false). In this way we don't need to alter the Program.cs

Let's also update the docs (please create an issue) with the steps to eventually start a Redis instance for testing those functionalities that require it

Reply:
Done